### ISSUE TITLE
Cannot run project on Android 8.0

### ISSUE THREAD

**Issue by janwiebe-jump** [infinitered/ignite#2719]
### Describe the bug

The app crashes immediately with an IllegalStateException:
Only fullscreen opaque activities can request orientation

This is caused by the portrait in the Android Manifest, that is set in the [app.json](https://github.com/infinitered/ignite/blob/master/boilerplate/app.json#L9):

See: https://stackoverflow.com/questions/48072438/java-lang-illegalstateexception-only-fullscreen-opaque-activities-can-request-o for reference.

Removing the orientation from the app.json solves this issue.

### Ignite version

7.15.0

### Additional info

just regular

**Comment by lindboe** [comment]
Hi, thanks for the report!

I think this is a bug with Expo: app.json is an Expo tool, and Expo does say Android 6+ should be supported: https://docs.expo.dev/versions/latest/config/app/#orientation.

That said, I don't think we're likely to remove the orientation configuration in order to support Android 8; older Android support isn't often a priority for us. Maybe we should raise the minSdkVersion in the template instead. @frankcalise do you have thoughts on this?

**Comment by frankcalise** [comment]
@lindboe Agree, if this is just configuration needed for a specific project, then it should just be that. Maybe a discussion thread or recipe page on better support for Android 8.0?

But looking at that stack overflow, is it possible to address such an issue with a specific version of styles.xml? Because we do have a config plugin setting `windowIsTranslucent` to address an Android 12 issue. 

![image](https://github.com/user-attachments/assets/6a88ab25-8d03-4757-87d9-1be8e6a7db69)

So maybe we can achieve this with a more tailored plugin?

https://github.com/infinitered/ignite/blob/master/boilerplate/plugins/withSplashScreen.ts#L62-L74

**Comment by frankcalise** [comment]
Closing this as this is more of a project configuration issue on a case by case basis, doesn't need to be in Ignite by default
